npm.yaml

```diff
@@ -43,14 +44,18 @@ pipeline:
              -name 'AUTHORS*' -o \
              -name 'LICENSE*' -o \
              -name 'license' -o \
+             -name 'node-gyp' -o \
              -name 'Makefile' -o \
              -name 'README*' -o \
              -name 'readme.markdown' \) -delete
 
       rm -rf ./*/.git* ./*/doc ./*/docs ./*/examples ./*/scripts ./*/test
-      rm -rf ./node-gyp/gyp/.git*
+      rm -rf ./node-gyp/
+      rm -rf node_modules/node-gyp
+      #rm -rf lib/node_modules/npm/node_modules/@npmcli/node-gyp/lib
+      #rm -rf lib/node_modules/npm/node_modules/@npmcli/run-script/lib/node-gyp-bin
```

Comment:
Should we delete these lines altogether, or leave a comment as well?

Reply:
let me delete this